lib/ts/recipe/thirdparty/components/themes/signInAndUp/providersForm.tsx

```diff
@@ -74,27 +74,24 @@ export const ThirdPartySignInAndUpProvidersForm: React.FC<SignInAndUpThemeProps>
                 buttonComponent: provider.getButton(),
             }));
         }
-        const tenantProviders = ThirdParty.tenantProviders || [];
+        const tenantProviders = Multitenancy.dynamicLoginMethods?.thirdparty.providers || [];
         const providers: { id: string; buttonComponent: JSX.Element }[] = [];
-        for (const tenantProvider in tenantProviders) {
+
+        for (const tenantProvider of tenantProviders) {
             let provider = props.providers.find((provider) => {
-                const { id } = tenantProviders[tenantProvider];
+                const { id } = tenantProvider;
                 return provider.id === id || provider.id.includes(id);
             });
             if (provider === undefined) {
                 provider = props.providers.find((provider) => {
-                    const { id } = tenantProviders[tenantProvider];
-                    return id.includes(provider.id);
+                    const { id } = tenantProvider;
+                    return id.startsWith(provider.id);
                 });
             }
             if (provider !== undefined) {
```

Comment:
This being undefined means that a provider was added to the tenant config that hasn't been configured on the frontend?